data/core/units/elves/Rider.cfg

```diff
@@ -18,9 +18,7 @@
     #mostly like a foot Elf, but they
     #aren't good at defending in villages
     #they are weak against piercing attacks
-    description= _ "The master horsemen of the elves are able to move through forests at a speed which would be suicidal for any human. Awe of the rider is quite justified — but questions have also been raised about the stock of the horses, for the feats they perform seem almost supernatural.
-
-This combination of incredible mobility and potent combat strength is one of the greatest assets the elves possess in warfare."
+   description= _ "The prevalence of guerrilla tactics among elvish strategy results in great emphasis on maintaining highly mobile units capable of fighting on an adaptable battlefront. This style of warfare requires rapid communication between small skirmish groups and the ability to quickly transfer troops between combat zones. Experienced Elvish riders know this well and thus devote most of their time on improving their horsemanship rather than their individual fighting skills. The speed at which they travel through the forest would be suicidal for any other cavalry, and they are tenacious enough to ride for as much as a couple days at a time without resting. While the intelligence and communication these horsemen provide is invaluable to Elven tactics, the lack of combat prowess and front line engagement means that Elvish riders are the oft unsung heroes of war, left supporting their brethren from the back lines."
```

Comment:
Pretty sure it should be "devote most of their time _to_", unless this is a dialectical variation.

Wouldn't "as much as two days" sound better?